### PR TITLE
refactor(protocol): use Bridge to send cross-chain owned contract transactions

### PR DESCRIPTION
**Pull request by dantaik**
This new implementation, compared with the one using signal service directly, does seem to be more clean and straight forward.

### CODE REVIEW

File: packages/protocol/contracts/L2/TaikoL2EIP1559Configurable.sol [hunk 1]
```diff
@@ -21,14 +21,7 @@ contract TaikoL2EIP1559Configurable is TaikoL2 {
...
     /// @param newGasExcess The new gas excess
-    function setConfigAndExcess(
-        Config memory config,
-        uint64 newGasExcess
-    )
-        external
-        virtual
-        onlyOwner
-    {
+    function setConfigAndExcess(Config memory config, uint64 newGasExcess) external {
```
Comment (adaki2004) [inline packages/protocol/contracts/L2/TaikoL2EIP1559Configurable.sol:24]: Do not we need `onlyOwner` here still ?

**Review by Brechtpd — APPROVED** [review]

**Comment by adaki2004** [comment]
I cannot comment on un-changed lines so i put it here.
Tests are failing because the `setConfigAndExcess` is onlyOwner now.

Fixes are (`TaikoL2.t.sol`):
kép

and

kép

**Comment by dantaik** [comment]
> setConfigAndExcess

Fixed

**Comment by dantaik** [comment]
@davidtaikocha could you fix issue in TaikoL2.ts, now the TaikoL2 contract needs the address manager's address, not the signal service's address in init().

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**